scraper/prototype.py

```diff
@@ -75,11 +74,11 @@ def export_data(centres_cherchés):
         }
         for code in import_departements()
     }
-    #center_list = []
-    #for centre in centres_cherchés:
-    #    center_list.append(centre)
-    #center_list = sort_centers(center_list)
+    center_list = []
     for centre in centres_cherchés:
+        center_list.append(centre)
```

Comment:
en fait le problème ici, c'est que tu crée un tableau de TOUS les centres en mémoire (plusieurs milliers, et ça va augmenter) parce que c'est une liste. donc la trier va beaucoup augmenter en temps.
Il vaudrait mieux trier  `par_departement[code_departement]['centres_indisponibles']` une fois qu'on a itéré sur tous les centres.